lib/SConscript

```diff
@@ -50,24 +50,20 @@ def build_config_template(target, source, env):
         handle.write(config_h)
 
 
-def prepare_c_source(text, escape_percentages=True):
+def prepare_c_source(text):
     # Prepare the Python source to be compatible with C-strings
     text = text.replace('"', '\\"')
-    text = text.replace('\\n', '\\\\n')
+    text = text.replace('\\', '\\\\')
```

Comment:
What's that reason for this change? It breaks compilation. The issue is also visible on Jenkins: https://travis-ci.org/github/sahib/rmlint/jobs/699617161

Reply:
The idea is to don't leak any escapes to C at all. I effed up when I let the quotes run first.